node/pkg/governor/devnet_config.go

```diff
@@ -18,16 +18,21 @@ func (gov *ChainGovernor) initDevnetConfig() ([]tokenConfigEntry, []tokenConfigE
 	}
 
 	flowCancelTokens := []tokenConfigEntry{}
+	flowCancelPipes := []pipe{}
 	if gov.flowCancelEnabled {
 		flowCancelTokens = []tokenConfigEntry{
 			{chain: 1, addr: "3b442cb3912157f13a933d0134282d032b5ffecd01a2dbf1b7790608df002ea7", symbol: "USDC", coinGeckoId: "usd-coin", decimals: 6, price: 1.001}, // Addr: 4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU, Notional: 6780118.197035182
 		}
+
+		flowCancelPipes = []pipe{
+			{first: vaa.ChainIDEthereum, second: vaa.ChainIDSui},
```

Comment:
Just a question. Does it make sense to have Sui here when the governor is not enabled for it in the devnet config?

Reply:
Probably not, thanks for catching this.